### PR TITLE
Linux 5.15.y/crocodile

### PR DESCRIPTION
**Pull request by lidaide**
I have test by these steps:
1. uname -a, it show "Linux crocodile 5.15.23-个1f8831b3194"
2. python3 wlan_ping.py will show OK
3. python3 bluetooth_scan.py will list bluetooth devices.

### CODE REVIEW

File: arch/arm64/configs/crocodile_defconfig [hunk 1]
```diff
@@ -190,21 +190,24 @@ CONFIG_CAN=m
...
+CONFIG_BT_HCIUART=y
+CONFIG_BT_HCIUART_BCSP=y
 CONFIG_BT_HCIUART_LL=y
 CONFIG_BT_HCIUART_BCM=y
 CONFIG_BT_HCIUART_QCA=y
 CONFIG_CFG80211=m
 CONFIG_MAC80211=m
 CONFIG_MAC80211_LEDS=y
-CONFIG_RFKILL=m
+CONFIG_RFKILL=y
```
Comment (htho-lgs) [inline arch/arm64/configs/crocodile_defconfig:211]: The description commit message is confusing:
- If you describe the configs, please refer to the help texts of them
- I does "we add the support to the Bluetooth device for MAYA-W1 serials, it was use High-Speed UART" mean?

**Review by htho-lgs — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/freescale/crocodile.dtsi [hunk 2]
```diff
@@ -593,6 +613,39 @@
...
+        fsl,pins = <
+            CROCODILE_IMX8_SD1_CLK		0x196
+            CROCODILE_IMX8_SD1_CMD		0x1d6
+            CROCODILE_IMX8_SD1_DATA0		0x1d6
+            CROCODILE_IMX8_SD1_DATA1		0x1d6
+            CROCODILE_IMX8_SD1_DATA2		0x1d6
+            CROCODILE_IMX8_SD1_DATA3		0x1d6
+        >;
+    };
+
```
[the remark this answers is not in the document]
Comment (lidaide) [reply arch/arm64/boot/dts/freescale/crocodile.dtsi:648]: have done by checkpath.pl tools.

File: arch/arm64/boot/dts/freescale/crocodile.dtsi [hunk 3]
```diff
@@ -45,6 +45,11 @@
 		enable-active-high;
 	};
 
+    usdhc1_pwrseq: usdhc1_pwrseq {
```
Comment (htho-lgs) [inline arch/arm64/boot/dts/freescale/crocodile.dtsi:48]: Still spaces

File: arch/arm64/boot/dts/freescale/crocodile.dtsi [hunk 4]
```diff
@@ -384,6 +389,21 @@
 	status = "okay";
 };
 
+&usdhc1 {
+    #address-cells = <1>;
+    #size-cells = <0>;
```
Comment (htho-lgs) [inline arch/arm64/boot/dts/freescale/crocodile.dtsi:394]: Spaces

**Review by htho-lgs — CHANGES_REQUESTED** [review]

File: arch/arm64/boot/dts/freescale/crocodile.dtsi (hunk 4 above)
Comment (lidaide) [reply arch/arm64/boot/dts/freescale/crocodile.dtsi]: have done

**Review by zandrey — APPROVED** [review]

**Pull request merged**